giveaways.html

```diff
@@ -151,13 +139,60 @@ <h4 class="leader">What does an active member of The Spatial Community mean?</h4
+                Palm Springs, CA, Washington DC, or Berlin!
+                <p></p>
+                Developer Summit is Esri's maker focused event series with roughly 4000 attendants annually. Come make new friends and learn about the newest technology directly from the devs. It's a conference for developers, by developers.
+        
+                <h3 class="leader">Prize Details</h3>
+                <h4>1st Place</h4>
+                <ul>
+                    <li>
+                        <h5>Esri DevSummit 2018 admission</h5>
+                        <div style="padding-left: 25px;">
+                            <strong>What is included?</strong><br>
+                            <ul>
+                                <li>Esri will cover your conference registration cost</li>
+                                <li>Feed you during the conference hours (continental breakfast on 'first come-first serve' availability and hosted lunch every day)</li>
+                            </ul>
+                            <strong>What isn't?</strong> Travel, lodging and dinners not included<br>
+                            <strong>Location?</strong> Palm Springs, DC, <strong>or</strong> Berlin, Germany<br>
+                            <strong>When is it?</strong> March 6-9, March 22, and October 23-25 (respectively)<br>
+                            <strong>Need more details?</strong> <a href="http://www.esri.com/events/devsummit">http://www.esri.com/events/devsummit</a><br>
+                        </div>
+                    </li>
+            
+                </ul>
+                <h4>2nd place</h4>
+                <ul>
+                    <li><h5>Python book 'Python Scripting for ArcGIS' (<a href="http://esripress.esri.com/display/index.cfm?fuseaction=display&websiteID=276&moduleID=0">link</a>)</h5></li>
+                    <li><h5>Geodev T-Shirt</h5></li>
+                    <li><h5>Esri Stickers</h5></li>
+                </ul>
+            </div>
+        </div>
+    </section>
+    
+    <!-- Esri DevSummit Section 2017-->
+    <section id="event2017_1" class="bg-light-gray">
         <div class="container large">
             <div class="row">
                 <div class="col-lg-12 text-center">
-                    <h2 class="section-heading">Giveaway #1 - Esri Developer Summit 2017</h2>
+                    <h2 class="section-heading">Esri Developer Summit 2017</h2>
                     <img src="https://upload.wikimedia.org/wikipedia/en/thumb/6/6e/Esri_logo.svg/1280px-Esri_logo.svg.png"width="250">
                     <h3 class="section-subheading text-muted">Esri sponsored Dev Summit Admission</h3>
                     <h3 class="text-center"><del>Giveaway Date will be January 27th, 2017 at 9 AM PST.</del></h3>
```

Comment:
Do we want to change these to past tense, and say who won?

Reply:
I think the year in the title works fine to imply past-tense, we also use the <del> tag to strike-through the old entries. Idk if we need to highlight the winner but I can be persuaded.